zync_maya.py

```diff
@@ -1269,6 +1269,7 @@ def submit(window):
           cmds.setAttr('vraySettings.hideRVOn', 1)
 
           cmds.setAttr('vraySettings.animBatchOnly', 0)
+          cmds.setAttr('defaultRenderGlobals.animation', 1)
```

Comment:
Done - added several comments to that section explaining the various options we set.